client/src/utilities/auth.js

```diff
@@ -1,100 +1,97 @@
 /* eslint-disable no-underscore-dangle */
 import Keycloak from 'keycloak-js';
-import Raven from 'raven-js';
-import queryString from 'querystringify';
-
+import Auth0 from 'auth0-js';
+import url from 'url';
+import * as k from './keycloak';
+import * as a0 from './auth0';
 import { get } from './api';
 
 let keycloak = null;
+let auth0 = null;
 let accessToken = null;
 
+function service() {
+  if (keycloak) {
+    return keycloak;
+  }
+  return auth0;
+}
+
+export function setKeycloak(K) {
+  keycloak = K;
+}
+
+export function setAuth0(A) {
```

Comment:
Unused?

Reply:
used in https://github.com/akvo/akvo-lumen/pull/2244/files#diff-4aa621b01f313081db8619cefcbe53feR106